tools/js-tools/eslintrc/base.js

```diff
@@ -168,5 +168,8 @@ module.exports = {
 
 		// Disabled pending #16099.
 		'inclusive-language/use-inclusive-words': 0,
+		// suppress errors for missing 'import React' in files
+		'react/react-in-jsx-scope': 'off',
```

Comment:
Should this really go here, or should it go in `tools/js-tools/eslintrc/react.js`?

Reply:
Nice catch! Thank you.